pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
 ]
-requires-python = ">=3.9"
+requires-python = ">=3.10"
```

Comment:
There's a place in README.md that says "requires python ≥ 3.9".  Can you update that place too?